src/Compilers/CSharp/Portable/Generated/CSharp.Generated.g4

```diff
@@ -955,6 +956,10 @@ type_of_expression
   : 'typeof' '(' type ')'
   ;
 
+with_expression
+  : expression syntax_token '{' (anonymous_object_member_declarator (',' anonymous_object_member_declarator)* ','?)? '}'
+  ;
```

Comment:
:) :)

We should consider if we can tweak the generator/xml to make it clear for the case of a token with a specified contextualkeyword type that that should emit better here :)

Reply:
Acutally, this is strange.  That should have worked:

```c#
        private static Production HandleTokenName(string tokenName)
            => GetSyntaxKind(tokenName) is var kind && kind == SyntaxKind.None ? RuleReference("SyntaxToken") :
               SyntaxFacts.GetText(kind) is var text && text != "" ? new Production(text == "'" ? "'\\''" : $"'{text}'") :
               tokenName.StartsWith("EndOf") ? new Production("") :
               tokenName.StartsWith("Omitted") ? new Production("/* epsilon */") : RuleReference(tokenName);
```

Should have produced `expression 'with' '{' ...`.  Hrmm...

Reply:
I think I just hadn't run generate-compiler-code again.